Common/src/main/java/com/tabnineCommon/capabilities/CapabilitiesService.java

```diff
@@ -55,8 +60,6 @@ private void fetchCapabilitiesLoop() {
     Optional<Long> lastPid = Optional.empty();
 
     try {
-      Thread.sleep(INITIAL_DELAY_MS);
-
```

Comment:
@amircodota we (@yairco1990 and i) decided to remove it, as it caused the chat to have a delay of 5-10 seconds on start, and without it its much better (idk what kind of math converts 2s delay to almost 10s delay, but 🤷🏻‍♂️ )

wdyt? should we worry for bad side effects?

Reply:
I honestly do not remember. Sorry :-)